### ISSUE TITLE
Explicit self-assignments in render management.

### ISSUE THREAD

**Issue by elfprince13** [GarageGames/Torque3D#1914]
Both of these appear to be the result of typos:
```
/Users/thomas/Torque3D/Engine/source/renderInstance/renderMeshMgr.cpp:248:28: Explicitly assigning value of variable of type 'GFXTextureObject *' to itself
/Users/thomas/Torque3D/Engine/source/renderInstance/renderPrePassMgr.cpp:483:28: Explicitly assigning value of variable of type 'GFXTextureObject *' to itself
```

Looking at both I see the same code:
```
            if ( passRI->accuTex != lastAccuTex )
            {
               sgData.accuTex = passRI->accuTex;
               lastAccuTex = lastAccuTex;
               dirty = true;
            }
```
From context, I believe the correct assignment is `lastAccuTex = passRI->accuTex`, can anyone confirm?

**Comment by Azaezel** [comment]
correct. batch support. intent is to skip reinitializing non-uniques.

**Comment by Areloch** [comment]
Corrected with #1921
